src/Host/Broker/Impl/Security/SecurityManager.cs

```diff
@@ -48,6 +49,34 @@ public class SecurityManager {
             return new ClaimsPrincipal(identity);
         }
 
+        private uint CreateProfile(string username, string domain, string password,out StringBuilder profilePath, out bool profileExists) {
+            uint error = 0;
+            profilePath = new StringBuilder();
+            profileExists = false;
+            using (NamedPipeClientStream client = new NamedPipeClientStream("RUserCreatorPipe"))
```

Comment:
Let's make it some kind of namespaced identifier (e.g. match the name of the assembly), or even use a GUID, just to make sure there's no collisions.